emantomo/__init__.py

```diff
@@ -25,164 +23,13 @@
 # *
 # **************************************************************************
 
-import os
-import subprocess
+import eman2
 
-import pwem
-import pyworkflow.utils as pwutils
-from scipion.install.funcs import VOID_TGZ
-
-import emantomo.constants as emanConst
-
-__version__ = "3.1.5"
+__version__ = "3.1.6"
```

Comment:
Should be 3.2.0

Reply:
You're right, it's a Minor one, not a patch.